### PR TITLE
Add English term "GNU Operating System"

### PR DESCRIPTION
**Pull request by jsteyn**
## Author: 

jsteyn

## Language: 

English

## Terms defined:

GNU Operating  System
- 
- 

## Editor

Assign the PR based on the language to the following person:

@zkamvar

### CODE REVIEW

**Comment by jsteyn** [comment]
Hi @zkamvar 
I did as you suggested in #114 and made a big bonfire out of it. Now I'm back again.
Could you perhaps tell me what the proper way of doing this is. (If there is such a thing). Per the instructions, I created the fork, cloned it, created a new branch, made the edits, add, commit and pushed it and then created a new PR. 
What should the next steps be if I want to work on the next edit?

Should I add an upstream? Should I fetch and merge (pull) it? Do I delete the branch I was working on?
 
Thank you very much for your help and it does help to know that I'm not the only one to royally mess things up!

**Comment by zkamvar** [comment]
> Hi @zkamvar
> I did as you suggested in #114 and made a big bonfire out of it. Now I'm back again.
> Could you perhaps tell me what the proper way of doing this is. (If there is such a thing). Per the instructions, I created the fork, cloned it, created a new branch, made the edits, add, commit and pushed it and then created a new PR.
> What should the next steps be if I want to work on the next edit?

That's a good question and something that I think needs to be added to the CONTRIBUTING document. 

The next steps are detailed in https://happygitwithr.com/upstream-changes.html#upstream-changes:

 1. make sure you have carpentries/glosario as your "upstream" repository
 1. make sure you are on the master branch of your repository
 1. use `git pull upstream master --ff-only`to update that branch
 1. check out a new branch, add your contribution, and create a new pull request

> 
> Should I add an upstream? Should I fetch and merge (pull) it? Do I delete the branch I was working on?
> 
> Thank you very much for your help and it does help to know that I'm not the only one to royally mess things up!

File: glossary.yml [hunk 1]
```diff
@@ -1869,6 +1869,15 @@
       [regular expressions](#regular_expression), such as `*.dat` to mean "all
       files whose names end in `.dat`". The name is derived from "global".
 
+- slug: gnu
```
Comment (zkamvar) [inline glossary.yml:1872]:
```suggestion
- slug: gnu
  ref:
    - gpl
```

File: glossary.yml [hunk 2]
```diff
@@ -1869,6 +1869,15 @@
...
+  en:
+    term: "GNU Operating System"
+    acronym: "GNU"
+    def: >
+      "GNU" is an operating system that is free software. GNU is a recursive acronym 
+      for "GNU is Not Unix!". The GNU operating system consists of GNU packages as well 
+      as free software realeased by third parties.
+
 - slug: gpl
```
Comment (zkamvar) [inline glossary.yml:1881]:
```suggestion
- slug: gpl
  ref:
    - gnu
```

**Review by zkamvar — APPROVED** [review]
Looks good to me. I added some suggested references, but it's not a deal breaker.

**Comment by jsteyn** [comment]
Hi @zkamvar. Thanks for all that. I accepted your changes and learnt about references at the same time. I've also added the upstream to my fork and did all the pulls and pushes, hops and jumps (it's like logging trees in the old days) required. Do I now delete the branch?

**Comment by zkamvar** [comment]
> Hi @zkamvar. Thanks for all that. I accepted your changes and learnt about references at the same time. I've also added the upstream to my fork and did all the pulls and pushes, hops and jumps (it's like logging trees in the old days) required. Do I now delete the branch?

Yes, you are correct that the branch should be deleted. And yes, the three-body problem with updating a fork is hella weird and it trips people up all the time.

**Pull request merged**